docker-compose.yml

```diff
@@ -1,13 +1,25 @@
-version: '3.7'
 services:
   citybase-flask-app:
+    profiles: [development]
```

Comment:
Going to use profiles to differentiate the style in which we want to run the code. Dev gets a flask deployment and production gets gunicorn one.